### DISCUSSION TITLE
How can I implement Tabs/Section within a  Tab/Section?

### DISCUSSION THREAD

**Discussion by Arnooodles** [csells/go_router#195] [Q&A]
Use Case:

Main Tabs
```
- /home
- /shops
- /profile
```
then in `/shops`, I have a sub-tabs/sections for each shop:
 example:
```
/shops/nike
/shops/apple
/shops/samsung
/shops/addidas
```
then in `/profile`
```
/profile/settings
/profile/orders
```

I have the following GoRouter :
```
final router = GoRouter(
  initialLocation: '/',
  urlPathStrategy: UrlPathStrategy.path,
  routes: [
    GoRoute(path: '/', redirect: (_) => '/home'),
    GoRoute(
        path: '/:section',
        pageBuilder: (context, state) {
          final section = baseState.params['section']!;
          return MaterialPage<void>(
            key: state.pageKey,
            child: BasePage(
              key: state.pageKey,
              section: section,
            ),
          );
        },
        ),
  ],
  errorPageBuilder: (context, state) => MaterialPage<void>(
    child: ErrorPage(
      error: state.error,
    ),
  ),
);
```
How should I proceed with adding the sub-routes?

**Comment by osaxma** [comment]
Did you check the [nested navigation example](https://github.com/csells/go_router/blob/master/example/lib/nested_nav.dart )? It's probably what you're looking for but instead of family tabs, you'd have shop tabs.

**Reply by Arnooodles** [reply]
Yes, I've checked that already but from what I understand is the sub-routes of that example are on a new page... what I want to achieve is something like this:
![image](https://user-images.githubusercontent.com/32187162/143522764-749e0dce-d366-46b0-a426-9b56af467988.png)

the main tab will be this:
![image](https://user-images.githubusercontent.com/32187162/143522851-a1135715-0c5e-4a50-ab77-c02a1b865966.png)

and the sub-routes:
![image](https://user-images.githubusercontent.com/32187162/143522992-39899f2c-d6e6-428e-99e3-ce006ea94f50.png)
